pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.7</version>
+    <version>3.51</version>
```

Comment:
The parent POM pulls in a lot of dependencies. Some of the dependencies of the POM we were using are incompatible with the current version of Maven.